.github/workflows/cfc-validation.yml

```
@@ -0,0 +1,299 @@
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Build project
+        run: |
+          echo "Building CFC Core..."
+          mvn clean install -DskipTests -B -V
+          echo "Build complete"
+
+      - name: Set up validation environment
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2016:info:5:8: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)